packages/xstate-test/test/index.test.ts

```diff
@@ -59,12 +59,12 @@ describe('events', () => {
     });
 
     const testModel = createTestModel(feedbackMachine, {
-      events: {
-        SUBMIT: { cases: [{ value: 'something' }, { value: '' }] }
+      eventCases: {
+        SUBMIT: [{ value: 'something' }, { value: '' }]
       }
     });
 
-    await testUtils.testModel(testModel);
+    await testUtils.testModel(testModel, {});
```

Comment:
Can this be optional?
```suggestion
    await testUtils.testModel(testModel);
```